hpx/runtime/components/server/runtime_support.hpp

```diff
@@ -160,10 +160,10 @@ namespace hpx { namespace components { namespace server
 
         /// \brief Actions to create new objects
         template <typename Component>
-        HPX_HOST_DEVICE naming::gid_type create_component();
+        naming::gid_type create_component();
 
```

Comment:
It doesn't break NVCC builds that was working before. I may be wrong but only the partitioned_vector.cu was using components and for the moment it has always fail at compilation.